akka-remote/src/main/scala/akka/remote/artery/ArteryTransport.scala

```diff
@@ -977,6 +977,16 @@ private[remote] class ArteryTransport(_system: ExtendedActorSystem, _provider: R
       envelopeBufferPool, giveUpAfter, createFlightRecorderEventSink()))
   }
 
+  private def aeronSink(outboundContext: OutboundContext, streamId: Int,
+                        bufferPool: EnvelopeBufferPool): Sink[EnvelopeBuffer, Future[Done]] = {
```

Comment:
Perhaps the other `private def aeronSink` should now just call this one with `envelopeBufferPool`?